css/styles.css

```diff
@@ -0,0 +1,320 @@
+
+  transition: color 250ms;
+}
+
+.nav__link:hover {
+  color: var(--color-text-blue);
+}
+
+.contacts {
+  display: flex;
+  gap: 50px;
+}
+
+.contacts--footer {
+  flex-direction: column;
+  gap: 9px;
+}
+
+.contacts__address {
+  font-style: normal;
+  color: var(--color-text-white);
+}
+
+.contacts__link {
+  display: flex;
+  align-items: center;
+
+  font-weight: 500;
+  font-size: 14px;
+
+  color: var(--color-text-gray);
+  transition: color 250ms;
+}
+
+.contacts__link:hover {
+  color: var(--color-text-blue);
+}
+
+.banner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  height: 600px;
+
+  background-color: var(--color-bg-dark-blue);
+}
+
+.banner__title {
+  margin: 0 0 30px;
+
+  width: 696px;
+
+  font-size: 44px;
+  font-weight: 900;
+  line-height: 60px;
+  text-align: center;
+  letter-spacing: 2.64px;
+
+  text-transform: uppercase;
+  color: var(--color-text-white);
+}
+
+.banner__button {
+  width: 216px;
+  height: 50px;
+
+  border: none;
+  border-radius: 4px;
+
+  color: #fff;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 30px;
+  letter-spacing: 0.96px;
+  background-color: var(--color-text-blue);
+}
+
+.features {
+  padding: 94px 0;
+}
+
+.features__list {
+  display: flex;
+  gap: 30px;
+}
+
+.features__item {
+  width: 270px;
+}
+
+.features__title {
+  margin: 0 0 10px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
+.features__text {
+  margin: 0;
+
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+
+  color: var(--color-text-gray);
+}
+
+.services {
+  padding: 0 0 94px;
+
+  text-align: center;
+}
+
+.services__title {
+  margin: 0 0 50px;
+
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 42px;
+  letter-spacing: 0.03em;
+}
+
+.services__list {
+  display: flex;
+  gap: 30px;
+}
+
+.services__item {
+  display: flex;
+}
+
+.services__image {
+  width: 370px;
+}
+
+.team {
+  padding: 94px 0 94px;
+
+  background-color: #f5f4fa;
+}
+
+.team__title {
+  margin: 0 0 50px;
+
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 42.19px;
+  letter-spacing: 0.03em;
+
+  text-align: center;
+}
+
+.team__list {
+  display: flex;
+  gap: 30px;
+}
+
+.person {
+  border-radius: 4px;
+  overflow: hidden;
+
+  box-shadow: 0px 2px 1px 0px #00000033;
+
+  box-shadow: 0px 1px 1px 0px #00000024;
+
+  box-shadow: 0px 1px 3px 0px #0000001f;
+}
+.person__image {
+  display: block;
+}
+.person__info {
+  padding: 30px 0;
+  background-color: #fff;
+
+  text-align: center;
+
+}
+.person__name {
+  margin: 0 0 10px;
+
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 18.75px;
+  letter-spacing: 0.03em;
+
+  color: var(--color-text-black);
+}
+.person__role {
+  margin: 0;
+
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 18.75px;
+  letter-spacing: 0.03em;
+
+  text-align: center;
+
+  color: var(--color-text-gray);
+}
+
+.footer {
+  margin: 0 auto;
```

Comment:
**`footer`** section does not need to be centered. **`container`** block is responsible for limiting the **`width`** of the content and making sure it's centered. 

Remove **`margin: 0 auto`** from the **`footer`** rules